packages/xo-server/src/xo-mixins/backups-ng/index.mjs

```diff
@@ -229,7 +229,7 @@ export default class BackupNg {
               }
             }),
           ])
-          if (Object.keys(remotes).length === 0) {
+          if (remoteIds.length > 0 && Object.keys(remotes).length === 0) {
```

Comment:
I believe this condition is necessary because the previous PR broke the implicit relation: `remoteIds = Object.keys(remotes)`.

Maybe it would be clearer to restore it?

Reply:
Ok, I've re-read your code and it's perfect, I will just add a small comment to make it clearer to me :slightly_smiling_face:

Reply:
In fact, I think we should still proceed if there are some other targets (SRs).

Reply:
And, ever more complicated, we should maybe proceed if rolling snapshots is enabled.

Maybe we should not throw an error and let the backup code decide whether it's abnormal or not.